scripts/core/ui/ui.ts

```diff
@@ -915,19 +927,31 @@ function splitterWidget(superdesk, $timeout, $rootScope) {
              */
             if (!authoring.length) {
                 $timeout(() => {
-                    authoring = element.next('#authoring-container');
                     authoring.width(superdesk.authoringWidth);
                 }, 0, false);
             }
 
             workspace.resizable({
                 handles: 'e',
-                minWidth: 400,
+                minWidth: MONITORING_MIN_WIDTH,
                 start: function(e, ui) {
-                    workspace.resizable({maxWidth: container.width() - 730});
+                    workspace.resizable({maxWidth: container.width() - AUTHORING_MIN_WIDTH});
                 },
                 resize: resize,
                 stop: afterResize,
+                create: () => {
+                    // On double click handle, reset size to default
+                    angular.element('.ui-resizable-handle').dblclick(() => {
+                        workspace.removeAttr('style');
```

Comment:
Can a specific style be removed instead of all inline styles?

Reply:
done :)